test/backend/all_test_names.txt

```diff
@@ -0,0 +1,874 @@
+from onnx 1.11.0
```

Comment:
Do you mind to include? If so, would add #automatically generated, do not modify... and maybe list the target that was use to create it?